templates/enterprise-license-clusterrole.yaml

```diff
@@ -20,6 +20,14 @@ rules:
     verbs:
       - get
 {{- end }}
+{{- if .Values.global.enablePodSecurityPolicies }}
+  - apiGroups: ["policy"]
+    resources: ["podsecuritypolicies"]
+    resourceNames:
+      - {{ template "consul.fullname" . }}-enterprise-license
+    verbs:
+      - use
+{{- end }}
```

Comment:
If `bootstrapACLs` and `enablePodSecurityPolicies` are both `false`, we'll have a cluster role without the `rules` key. I don't think it's wrong, and Kube won't complain about it. Just something to be aware of because in other places, for example in [`client-clusterrole.yaml`](https://github.com/hashicorp/consul-helm/blob/9cc2e9d58a632eecb9ec419a151fcf0605fed57b/templates/client-clusterrole.yaml#L31), we're setting it an empty array rather than leaving it out.

Reply:
fixed